stellar-multitenant/internal/validators/tenant_validator.go

```diff
@@ -60,6 +62,15 @@ func (tv *TenantValidator) ValidateCreateTenantRequest(reqBody *TenantRequest) *
 	tv.Check(reqBody.OwnerLastName != "", "owner_last_name", "owner_last_name is required")
 	tv.Check(reqBody.OrganizationName != "", "organization_name", "organization_name is required")
 
+	tv.Check(reqBody.DistributionAccountType != "", "distribution_account_type", "distribution_account_type is required")
+	if reqBody.DistributionAccountType != "" {
+		switch schema.AccountType(reqBody.DistributionAccountType) {
+		case schema.DistributionAccountStellarEnv, schema.DistributionAccountStellarDBVault, schema.DistributionAccountCircleDBVault:
+		default:
+			tv.Check(false, "distribution_account_type", "invalid distribution account type")
```

Comment:
👋 Can you update this error message with the list of allowed types and the name of the provided type? This would make a better experience for the Host when they need to debug this error.